export/httpQuery.ts

```diff
@@ -1,9 +1,11 @@
 import { parse } from '../shims/url';
 import { Socket } from '../shims/net';
-import { types } from '.';
+import { types as defaultTypes } from '.';
 
 // @ts-ignore -- this isn't officially exported by pg
 import { prepareValue } from '../node_modules/pg/lib/utils';
+// @ts-ignore -- this isn't officially exported by pg
+import TypeOverrides from '../node_modules/pg/lib/type-overrides';
```

Comment:
I was wondering - why aren't we just doing the following? 

```suggestion
// @ts-ignore -- this isn't officially exported by pg
import { prepareValue } from 'pg/lib/utils';
// @ts-ignore -- this isn't officially exported by pg
import TypeOverrides from 'pg/lib/type-overrides';
```

Reply:
Right, that's definitely better: I've changed it!